testsuite/cluster-test/src/cluster_builder.rs

```diff
@@ -129,8 +126,9 @@ impl ClusterBuilder {
         image_tag: &str,
         config_overrides: &[String],
         delete_data: bool,
-    ) -> Result<Vec<Instance>> {
+    ) -> Result<(Vec<Instance>, (Vec<Instance>, Vec<Instance>))> {
```

Comment:
Is there benefit of using double nested tuple like `(vec, (vec, vec))`, instead of `(vec, vec, vec)`?

Reply:
My thought is logically combine vault and LSR together in a nested tuple. I can flat them if you think that is redundant

Reply:
Yeah, I would rather flatten them